### PR TITLE
Added RundBundle

### PR DESCRIPTION
**Pull request by Rutik7066**
/claim #875 
RunBundle Added

### CODE REVIEW

**Comment by Rutik7066** [comment]
Hi @tolgaOzen implemented runbundle. If any changes needed, let me know.

**Comment by tolgaOzen** [comment]
Hello @Rutik7066 , I did a review and couldn't see the tests, can you add them?

File: internal/storage/memory/tracer.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package memory
+
+import (
+	"go.opentelemetry.io/otel"
+)
+
+var tracer = otel.Tracer("storage.postgres")
```
Comment (tolgaOzen) [inline internal/storage/memory/tracer.go:7]: It should be `var tracer = otel.Tracer("storage.memory")`. Could you make this change?

**Comment by Rutik7066** [comment]
Hi @tolgaOzen sure i will add the test

File: internal/storage/memory/tracer.go (hunk 1 above)
Comment (Rutik7066) [reply internal/storage/memory/tracer.go:7]: done

**Pull request closed without merging**